src/components/Table/GridTable.tsx

```diff
@@ -174,6 +174,7 @@ export type GridSortConfig<S> =
       /** The optional initial column (index in columns) and direction to sort. */
       initial?: [S | GridColumn<any>, Direction] | undefined;
       caseSensitive?: boolean;
+      persistent?: S | undefined;
```

Comment:
I think technically would want to do:

```
[S | GridColumn<any>, Direction]
```

So that you could say `[1, "ASC"]` i.e. to put favorites first (or last or what not).

Reply:
I went back and forth on this,

it is now working with the desired, without allowing the persistent column to be moved to the bottom.

There is not currently a use case for putting favorites last, however this could be added, just more cycles on this.